extensions/iframe/common/client.mjs

```diff
@@ -14,7 +14,8 @@ class IframeOverlay extends Overlay {
 
     const expandBtn  = document.getElementById('iframe-expand');
     expandBtn.onclick = () => {
-      mainUI.enterFocusMode(this, OverlayPosition.LEFT_BOTTOM);
```

Comment:
Maybe add a TODO to note on better handling for hiding of userOverlay. Or just modify `enterFocusMode()` to handle hiding of userOverlay?